opendebates/router.py

```diff
@@ -75,7 +76,7 @@ def was_db_written():
     return state.was_written()
 
 
-class DBRoutingMiddleware(object):
+class DBRoutingMiddleware(MiddlewareMixin, object):
```

Comment:
If we're inheriting from MiddlewareMixin (or anything), we don't also need to inherit from object.